client/blocks/plugin-scheduled-updates-common/styles.scss

```diff
@@ -84,7 +84,7 @@ $brand-display: "SF Pro Display", sans-serif;
 
 		p {
 			color: var(--studio-gray-80);
-			margin: 1.5rem auto;
+			margin-block: 1.5rem;
```

Comment:
The horizontal `auto` padding wasn't needed for anything and it was making the text centre unexpectedly.